client-course-schedulizer/src/utilities/helpers/caseFunctions.ts

```diff
@@ -333,7 +335,7 @@ export const prefixCase = (value: string): string[] => {
   return value.replace(" ", "").split(/[;,\n]/);
 };
 
-export const numberDefaultZeroCase = (value: string): number => {
+export const integerDefaultZeroCase = (value: string): number => {
```

Comment:
I changed the name of this function to specify that it should only be used for integers. It was the source of the fractional loads bug.